jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -37,20 +37,34 @@ MlirToJlmConverter::ReadAndConvertMlir(const util::filepath & filePath)
   {
     JLM_ASSERT("Parsing MLIR input file failed.");
   }
-  return ConvertMlir(block);
+  auto & topNode = block->front();
+  auto omegaNode = ::mlir::dyn_cast<::mlir::rvsdg::OmegaNode>(topNode);
+  return ConvertOmega(omegaNode);
```

Comment:
These four lines look like they do the same as ConvertMlir. Can it be called instead?